Dockerfile

```diff
@@ -22,9 +22,6 @@ ARG UBUNTU_DATE
 # Docker build debug logging, green colored
 RUN printf "\033[1;32mFROM ubuntu:${UBUNTU_FLAVOR}-${UBUNTU_DATE} \033[0m\n"
 
-MAINTAINER Diego Molina <diemol@gmail.com>
```

Comment:
maybe don't remove the maintainers:)

Reply:
omg I don't know why this even exists. sorry :pray:

Reply:
Ah I remember now, I forgot to add it as a LABEL. the MAINTAINER TAG is depricated, that's why I removed it. well i forgot to replace it...

Reply:
I removed you originally from MAINTAINER tag because it's depricated. LABEL is the appropiate way to do it. Sorry wasn't my intention to remove you as an MAINTAINER.

The LABEL can be found in line 27

Reply:
I removed you originally from MAINTAINER tag because it's depricated. LABEL is the appropiate way to do it. Sorry wasn't my intention to remove you as an MAINTAINER.

The LABEL can be found in line 27